chemical fingerprints (x)

GLOSSARY:

---------

An assay is an investigative (analytic) procedure in laboratory medicine, mining, pharmacology, environmental biology and molecular biology for qualitatively assessing or quantitatively measuring the presence, amount, or functional activity of a target entity. The analyte can be a drug, biochemical substance, chemical element or compound, or cell in an organism or organic sample. The measured entity is often called the analyte, the measurand, or the target of the assay. An assay usually aims to measure an analyte's intensive property and express it in the relevant measurement unit (e.g. molarity, density, functional activity in enzyme international units, degree of effect in comparison to a standard, etc.). 

bioinformatics:  an  interdisciplinary  field of biomedical engineering and biotechnology that incorporates  computational,  mathematical,  statistical analysis and information technology to provide a better solution to biological and biomedical engineering problems

biological target: anything within a living organism to which some other entity (like a ligand or drug) is directed or binds resulting in a change in its behavior or function

chemical library or compound library is a collection of stored chemicals usually used ultimately in high-throughput screening or industrial manufacture. The chemical library can consist in simple terms of a series of stored chemicals. Each chemical has associated information stored in some kind of database with information such as the chemical structure, purity, quantity, and physiochemical characteristics of the compound; In drug discovery high-throughput screening, it is desirable to screen a drug target against a selection of chemicals that try to take advantage of as much of the appropriate chemical space as possible. The chemical space of all possible chemical structures is extraordinarily large. Most stored chemical libraries do not typically have a fully represented or sampled chemical space mostly because of storage and cost concerns. However, since many molecular interactions cannot be predicted, the wider the chemical space that is sampled by the chemical library, the better the chance that high-throughput screening will find a "hit"—a chemical with an appropriate interaction in a biological model that might be developed into a drug. 

Chemical space is a concept in cheminformatics referring to the property space spanned by all possible molecules and chemical compounds adhering to a given set of construction principles and boundary conditions. It contains millions of compounds which are readily accessible and available to researchers. 


drug discovery is the process by which new candidate medications are discovered; historically, drugs were discovered by identifying the active ingredient from traditional remedies or by serendipitous discovery, as with penicillin. More recently, chemical libraries of synthetic small molecules, natural products or extracts were screened in intact cells or whole organisms to identify substances that had a desirable therapeutic effect in a process known as classical pharmacology. After sequencing of the human genome allowed rapid cloning and synthesis of large quantities of purified proteins, it has become common practice to use high throughput screening of large compounds libraries against isolated biological targets which are hypothesized to be disease-modifying in a process known as reverse pharmacology. Hits from these screens are then tested in cells and then in animals for efficacy.
  Modern drug discovery involves the identification of screening hits, medicinal chemistry and optimization of those hits to increase the affinity, selectivity (to reduce the potential of side effects), efficacy/potency, metabolic stability (to increase the half-life), and oral bioavailability. Once a compound that fulfills all of these requirements has been identified, the process of drug development can continue. If successful, clinical trials are developed.[5]Modern drug discovery is thus usually a capital-intensive process that involves large investments by pharmaceutical industry corporations as well as national governments (who provide grants and loan guarantees). Despite advances in technology and understanding of biological systems, drug discovery is still a lengthy, "expensive, difficult, and inefficient process" with low rate of new therapeutic discovery.[6] In 2010, the research and development cost of each new molecular entity was about US$1.8 billion.[7] In the 21st century, basic discovery research is funded primarily by governments and by philanthropic organizations, while late-stage development is funded primarily by pharmaceutical companies or venture capitalists.[8] To be allowed to come to market, drugs must undergo several successful phases of clinical trials, and pass through a new drug approval process, called the New Drug Application in the United States. 

High-throughput screening (HTS) is a method for scientific experimentation especially used in drug discovery and relevant to the fields of biology and chemistry. Using robotics, data processing/control software, liquid handling devices, and sensitive detectors, high-throughput screening allows a researcher to quickly conduct millions of chemical, genetic, or pharmacological tests. Through this process one can rapidly identify active compounds, antibodies, or genes that modulate a particular biomolecular pathway. The results of these experiments provide starting points for drug design and for understanding the noninteraction or role of a particular location.

target proteins: functional biomolecules addressed and controlled by biologically active compounds; identification of target proteins, the investigation of signal transduction processes, and the understanding of their interaction with ligands are key elements of modern biomedical research; interaction with target proteins is the molecular origin of most drugs

IC50: half maximal inhibitory concentration; a measure of the potency of a substance in inhibiting a specific biological or biochemical function; indicates how much of a particular inhibitory substance (e.g. drug) is needed to inhibit, in vitro, a given biological process or biological component by 50% (target variable); expressed as molar concentration.; commonly used as a measure of antagonist drug potency in pharmacological research; comparable to EC50 for excitatory (agonist) drugs, EC50 represents the dose or plasma concentration required for obtaining 50% of a maximum effect; IC50 can be determined with functional assays or with competition binding assays; sometimes, IC50 values are converted to the pIC50 scale.

    pIC 50 = − log 10 ⁡ ( IC 50 ) {\displaystyle {\ce {pIC_{50}}}=-\log _{10}{\ce {(IC_{50})}}} {\displaystyle {\ce {pIC_{50}}}=-\log _{10}{\ce {(IC_{50})}}}

where, due to the minus sign, higher values of pIC50 indicate exponentially more potent inhibitors; pIC50 is usually given in terms of molar concentration (mol/L, or M),

Lipinski's rule of five (Pfizer's rule of five, Lipinski descriptors, RO5): heuristic to evaluate druglikeness, i.e. determine if a chemical compound with a certain pharmacological or biological activity has chemical properties and physical properties that would make it a likely orally active drug in humans; candidate drugs that conform to the RO5 tend to have lower attrition rates during clinical trials and hence have an increased chance of reaching the market; rule formulated by Christopher Lipinski based on the observation that most orally administered drugs are relatively small and moderately lipophilic molecules.

QSARs (quantitative structure-activity relationship models)
: used to predict the physicochemical, biological, and environmental fate properties of compounds from the knowledge of their chemical structure (often decision trees or ANNs)

quantitative biology: an umbrella term encompassing the use of mathematical, statistical or computational techniques to study life and living organisms; the central theme and goal of quantitative biology is the creation of predictive models based on fundamental principles governing living systems

The simplified molecular-input line-entry system (SMILES) is a specification in the form of a line notation for describing the structure of chemical species using short ASCII strings. SMILES strings can be imported by most molecule editors for conversion back into two-dimensional drawings or three-dimensional models of the molecules.

---------------

Bioinformatics Project: Drug Discovery



